pkg/app/piped/cloudprovider/cloudrun/state.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2022 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cloudrun
+
+import (
+	"sort"
+	"time"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/pipe-cd/pipecd/pkg/model"
+)
+
+func MakeResourceStates(svc *Service, revs []*Revision, now time.Time) []*model.CloudRunResourceState {
+	states := make([]*model.CloudRunResourceState, 0, len(revs)+1)
+
+	// Set service state.
+	sm, err := svc.ServiceManifest()
+	if err == nil {
+		status, desc := svc.HealthStatus()
+		states = append(states, makeResourceState(sm.u, status, desc, now))
+	}
+
+	// Set active revision states.
+	for _, r := range revs {
+		rm, err := r.RevisionManifest()
+		if err != nil {
+			continue
+		}
+
+		var (
+			status, desc = r.HealthStatus()
+			state        = makeResourceState(rm.u, status, desc, now)
+		)
+		states = append(states, state)
```

Comment:
```suggestion
		status, desc := r.HealthStatus()
		states = append(states, makeResourceState(rm.u, status, desc, now))
```
nits, not a rule or something, but I think `var ()` should only be used in case we want some default value for that variable and change it later if there is a value (not default) that should be assigned to that var, of course, it's up to you, so it's just a nits

Reply:
Certainly, You're right.
I fixed above by https://github.com/pipe-cd/pipecd/pull/3238/commits/2f4e7482e72e4aa7c3aea4c134d12d506f6beb27